workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/MonoBehaviourActivationManager.cs

```diff
@@ -105,25 +105,48 @@ public void EnableSpatialOSBehaviours()
             foreach (var behaviour in behavioursToEnable)
             {
                 behaviour.enabled = true;
```

Comment:
Can exceptions propagate out of the `OnEnable` method of the MonoBehaviour?

Reply:
Thanks for catching! Fixed.